spk/firebird/src/service-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
```

Comment:
the shebang must be removed.
This file is never run as standalone script. It is embedded into `scripts/service-setup` and this one is sourced into other scripts.
```suggestion
```

Reply:
Done.